cpp/include/raft/random/rng.hpp

```diff
@@ -298,8 +301,8 @@ class Rng : public detail::RngImpl {
    * @param sigma the sigma
    * @param stream stream where to launch the kernel
    */
-  template <typename Type, typename LenType = int>
-  void rayleigh(Type* ptr, LenType len, Type sigma, cudaStream_t stream)
+  template <typename OutType, typename LenType = int>
```

Comment:
Any particular reason for the name change?

Reply:
Just for consistency with other functions.